sriov/common/exec.py

```diff
@@ -109,13 +150,17 @@ def stop_testpmd(self):
         # sleep before return
         time.sleep(1)
         return exit_status
-        
+       
     def testpmd_cmd(self, cmd):
-        """send cmd in the testpmd session
-        
-        :param cmd: the command to be executed in the testpmd session
-        
-        :return: True on success
+        """ Send a command to the TestPMD application 
+
+        Args:
+            self:      self
+            cmd (str): the command to be executed in the TestPMD session
+
+        Returns:
+            False:           TestPMD is not active
```

Comment:
`False` as the return value here to mean the underlying `testpmd` is not active doesn't make semantic sense to me.

`testpmd_cmd()` should throw an exception if the caller attempts to send a command when there is no `testpmd` to send it to. Ideally different Exceptions if the `testpmd` is not yet started, has been stopped, and if it has been disconnected somehow.
There's another method in the interface of the class to check if a `testpmd` is active (`testpmd_active()`) if that's what the caller is interested in so this method should not share that responsibility (though it can absolutely use the `testpmd_active()` method to check if an exception is warranted).